scripts/envhook.py

```diff
@@ -28,9 +28,16 @@ def main(argv):
             link_dir = os.path.abspath(os.path.join(os.path.dirname(site.__file__), 'site-packages'))
     else:
         raise RuntimeError('Need to be run from within a virtualenv')
-    link = os.path.join(link_dir, 'sitecustomize.py')
     dst = os.path.abspath(__file__)
-    dst = os.path.relpath(dst, link_dir)
+    try:
+        import sitecustomize
+    except ImportError:
+        pass
+    else:
+        if os.path.realpath(sitecustomize.__file__) != dst:
+            raise RuntimeError(f'A different sitecustomize.py already exists at {sitecustomize.__file__}')
```

Comment:
```suggestion
            raise RuntimeError(f'A different sitecustomize module already exists at {sitecustomize.__file__}')
```

Both `sitecustomize.py` and `sitecustomize/__init__.py` can be imported using `import sitecustomize`, IIRC.

Reply:
Good point.